src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -329,12 +330,32 @@
       <TestGroup Condition=" $([MSBuild]::ValueOrDefault(%(XUnitWrapperDlls), '').IndexOf('.XUnitWrapper.dll')) != $([MSBuild]::ValueOrDefault(%(XUnitWrapperDlls), '').LastIndexOf('.XUnitWrapper.dll')) "></TestGroup>
     </Payloads>
 
-    <HelixWorkItem Include="@(Payloads->Metadata('PayloadGroup'))">
-      <PayloadDirectory>%(PayloadDirectory)</PayloadDirectory>
-      <Command>dotnet $(XUnitRunnerDll) %(XUnitWrapperDlls) $(XUnitRunnerArgs)</Command>
-      <Command Condition=" '%(TestGroup)' != '' ">dotnet $(XUnitRunnerDll) %(XUnitWrapperDlls) $(XUnitRunnerArgs) -trait TestGroup=%(TestGroup)</Command>
-      <Timeout Condition=" '$(TimeoutPerTestCollectionInMinutes)' != '' ">$([System.TimeSpan]::FromMinutes($(TimeoutPerTestCollectionInMinutes)))</Timeout>
-    </HelixWorkItem>
+    <PropertyGroup>
```

Comment:
It seems that `PropertyGroup` should be one level higher

Reply:
@premun  - whenever you have a moment, what does this suggestion indicate, does the entire part between <PropertyGroup>  and </PropertyGroup> needs to be within the actual Payload ?

Reply:
I think Katya meant that the `ItemGroup` above starting on line 316 is not closed properly?

Reply:
You just can't have a property group inside an item group, right?

Reply:
Yes, that is the issue